staging/src/k8s.io/client-go/util/workqueue/queue.go

```diff
@@ -33,6 +33,46 @@ type Interface interface {
 	ShuttingDown() bool
 }
 
+// Queue is the underlying storage for items.
+type Queue interface {
+	// Touch marks an existing item when it is added again.
```

Comment:
what does it means semantically? I assume it will keep the object in the queue, but might change the position in the queue?

Reply:
Yeah, it basically says the same item while still in the queue is now being added again. For the implementations which can change the priority (or position in your term), this function can be hooked.